lib/src/models/soundboard/soundboard.dart

```diff
@@ -10,6 +12,16 @@ class PartialSoundboardSound extends WritableSnowflakeEntity<SoundboardSound> {
   final SoundboardManager manager;
 
   PartialSoundboardSound({required super.id, required this.manager});
+
+  /// The sound asset for this soundboard sound.
+  CdnAsset get sound => CdnAsset(
+        client: manager.client,
+        base: HttpRoute()..soundboardSounds(),
+        // We can't use .soundboardSounds(id: id.toString()) because hash would be empty, adding an empty string will populate
+        // `route.segments` which will join it with a trailing slash and Discord doesn't handle trailing slashes.
```

Comment:
Can we instead fix HttpRoute to filter out empty segments before joining (i.e not add a trailing slash, since that's technically a bug)?

Alternatively fix CdnAsset to check for an empty hash. Though overall I think we should leave a TODO to make hash nullable in the next major release.